resources/js/src/app/components/customer/AddressInputGroup.js

```diff
@@ -116,6 +136,35 @@ Vue.component("address-input-group", {
         emitInputEvent(field, value)
         {
             this.$emit("input", { field, value });
+        },
+
+        isInOptionalFields(locale, key)
+        {
+            return this.optionalAddressFields[locale].includes(key);
+        },
+
+        isInRequiredFields(locale, key)
+        {
+            return (this.requiredAddressFields && this.requiredAddressFields[locale] && this.requiredAddressFields[locale].includes(key));
+        }
+    },
+
+    filters: {
+        transformRequiredLabel(label, shouldMarkRequired)
+        {
+            return shouldMarkRequired ? label + "*" : label;
+        }
+    },
+
+    directives: {
+        "data-validate": {
```

Comment:
Schöner in der Verwendung wäre vielleicht folgende Struktur:
`v-data-validate:TYPE="CONDITION"`

Die Abfrage wäre dann:
```
if ( binding.value )
{
  el.dataset.validate = binding.argument;
}
```

In diesem Fall könnte man das ganze auch gut als globale Direktive bereitstellen